src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -28,21 +28,21 @@ class JsonAdaptedPerson {
     private final String phone;
     private final String email;
     private final String address;
-    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final List<JsonAdaptedTag> tags = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedPerson} with the given person details.
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("tagged") List<JsonAdaptedTag> tags) {
```

Comment:
Should we change both instances of `tagged` in this line?

Reply:
Hi Prof, I have tried to change both instances of `tagged` in this line when i was fixing this bug. But changing the `@JsonProperty` tag will affect other files as well, as `tagged` is currently being used in those files.